dpnp/linalg/dpnp_iface_linalg.py

```diff
@@ -914,27 +914,33 @@ def lu_factor(a, overwrite_a=False, check_finite=True):
     where `P` is a permutation matrix, `L` is lower triangular with unit
     diagonal elements, and `U` is upper triangular.
 
+    For full documentation refer to :obj:`scipy.linalg.lu_factor`.
+
     Parameters
     ----------
-    a : (M, N) {dpnp.ndarray, usm_ndarray}
+    a : (..., M, N) {dpnp.ndarray, usm_ndarray}
         Input array to decompose.
     overwrite_a : {None, bool}, optional
-        Whether to overwrite data in `a` (may increase performance)
+        Whether to overwrite data in `a` (may increase performance).
+
         Default: ``False``.
     check_finite : {None, bool}, optional
         Whether to check that the input matrix contains only finite numbers.
         Disabling may give a performance gain, but may result in problems
         (crashes, non-termination) if the inputs do contain infinities or NaNs.
 
+        Default: ``True``.
+
     Returns
     -------
-    lu :(M, N) dpnp.ndarray
-        Matrix containing U in its upper triangle, and L in its lower triangle.
-        The unit diagonal elements of L are not stored.
-    piv (K, ): dpnp.ndarray
-        Pivot indices representing the permutation matrix P:
+    lu : (..., M, N) dpnp.ndarray
+        Matrix containing `U` in its upper triangle,
+        and `L` in its lower triangle.
+        The unit diagonal elements of `L` are not stored.
+    piv : (..., K) dpnp.ndarray
+        Pivot indices representing the permutation matrix `P`:
         row i of matrix was interchanged with row piv[i].
-        ``K = min(M, N)``.
+        Where ``K = min(M, N)``.
 
```

Comment:
The below limitation `Only two-dimensional input matrices are supported.` is still there. Is that expected?